### PR TITLE
Add ar.yml 

### PR DESCRIPTION
**Pull request by 0xGhada**
# Description

## Corresponding Issue #2177 

# Screenshots

---

Reviewing this pull request? Check out our [Code Review Practices](https://github.com/ifmeorg/ifme/wiki/Code-Review-Practices) guide if you haven't already!

### CODE REVIEW

**Comment by julianguyen** [comment]
Should this PR title be renamed to something related to translations?

**Comment by julianguyen** [comment]
I think we should rename the title of this PR!

**Comment by puregeek** [comment]
@julianguyen What should I name it.. I just found the most similar files named as well so I did

File: config/locales/ar.yml [hunk 1]
```diff
@@ -0,0 +1,1039 @@
+en:
```
Comment (julianguyen) [inline config/locales/ar.yml:1]: You'll want to rename this to `ar:`

And then for the right-hand text on each line below, you'll want to translate the English text.

**Pull request closed without merging**